File: src/main/java/com/mgjg/kmztracker/example/NotificationView.java
package com.mgjg.kmztracker.example;

/**
 * Created by marianne on 6/21/2015.
 */

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

import com.mgjg.kmztracker.R;

public class NotificationView extends Activity
{
    public void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.notification);

        TextView tv = (TextView) findViewById(R.id.tv_notification);
        Bundle data = getIntent().getExtras();
        tv.setText(data.getString("content"));
    }
}

File: src/main/java/com/mgjg/kmztracker/example/NotificationView.kt
package com.mgjg.kmztracker.example

/**
 * Created by marianne on 6/21/2015.
 */

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

import com.mgjg.kmztracker.R

class NotificationView : Activity() {
  public override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.notification)

    val tv = findViewById(R.id.tv_notification) as TextView
    val data = intent.extras
    tv.text = data!!.getString("content")
  }
}